### PR TITLE
Changed tests to use chai, chai and mocha now in package.json

### PR DESCRIPTION
**Pull request by aowenmeehanwork**
Tests now check to see if data coming in from the API is valid

### CODE REVIEW

**Review by clarebuckley — APPROVED** [review]
All tests pass

File: src/backend-api/test/api_tests.js [hunk 1]
```diff
@@ -26,7 +40,19 @@ describe('Getting all job families', function() {
             if (err) console.log(err);
```
Comment (micalexis) [inline src/backend-api/test/api_tests.js:40]: wrap in brackets

File: src/backend-api/test/api_tests.js [hunk 2]
```diff
@@ -26,7 +40,19 @@ describe('Getting all job families', function() {
...
-            assert.equal(res.statusCode, 200);
+            expect(res.statusCode).to.equal(200);
+        });
+    });
+
+    it('Should return columns with names job_family_id and job_family_name', function() {
+    request.get({
+        uri: API_URL + 'families', 
+    }, (err, res, body) => {
+        if (err) console.log(err);
```
Comment (micalexis) [inline src/backend-api/test/api_tests.js:51]: same as above

**Review by micalexis — CHANGES_REQUESTED** [review]
address the comments

**Comment by aowenmeehanwork** [comment]
Outdated pull request, this functionality is implemented in US011/view-job-family-capability with appropriate requested changes to logging. Closing pull request now.

**Pull request closed without merging**